orchagent/flex_counter/flex_counter_manager.h

```diff
@@ -54,6 +61,26 @@ class FlexCounterManager
                 const std::unordered_set<std::string>& counter_stats);
         void clearCounterIdList(const sai_object_id_t object_id);
 
+        std::string getGroupName()
```

Comment:
Suggest to use ```const``` keyword for all getters to make them constant functions.
It is not required but recommended, since getters purpose is only to retrieve data and not to modify anything.

Reply:
Modified